rpc/net.go

```diff
@@ -17,7 +17,7 @@ import (
 
 const defaultSearchTime = time.Second * 12
 
-// Net contains the functions required by the rpc service into the network.
+// Net contains the functions required by the rpc service into the network. TODO: "into the network"???
```

Comment:
I didn't understand the comment.

Reply:
```suggestion
// Net contains the network-related functions required by the rpc service.
```

maybe something like this?